openc3/lib/openc3/api/cmd_api.rb

```diff
@@ -314,9 +314,19 @@ def get_cmd_cnts(target_commands, scope: $openc3_scope, token: $openc3_token)
     # PRIVATE implementation details
     ###########################################################################
 
-    def cmd_implementation(method_name, *args, range_check:, hazardous_check:, raw:, timeout: nil,
+    def cmd_implementation(method_name, *args, range_check:, hazardous_check:, raw:, timeout: nil, log_message: nil,
```

Comment:
I'm open to other names but I didn't want to use just `log` because we're still logging ... it's just the log message which we're either outputting or surpressing.